src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -92,6 +92,19 @@ public function iBrowseProductsFromTaxon(?TaxonInterface $taxon = null): void
         }
     }
 
+    /**
+     * @When I browse products from product taxon code :taxon
+     */
+    public function iBrowseProductsFromProductTaxonCode(?TaxonInterface $taxon = null): void
```

Comment:
```suggestion
    public function iBrowseProductsFromProductTaxonCode(TaxonInterface $taxon): void
```
Is there any case when it can be null? :thinking: